src/app/project-dialog/project-dialog.component.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../assets/stylesheets/color-variables";
+@import "../../assets/stylesheets/custom";
+.createProjectBlock{
+    font-size: rem(10);
+    padding: 0;
+    > div{
```

Comment:
It would be nice if you write style by giving some class to html tag. Instead of writing style directly to tag. mostly for these reason:
1. we can use same style for different element if needed.
2. if we add same element in future then same style will apply to that, even if we don't need that.